Navbar updated

GerritBurgerDev opened the pull request.

lrparkinson commented:
I modified the nav bar myself a bit along with the grid layout in my front-end-auth branch since I removed the login page: If we only gona have google auth then no page need. I refactored index.html to include the navbar and allowed it to be passed a slot which will be placed at the end of the nav bar. At the moment the google sign in button and welcome message (for logged in users) are set there. I was thinking we could get my front-end-auth branch merged first and the just update your nav-bar branch from master and realllign the new navbar styling you implement. Maybe even place the search bar you made in the nav in the center?

GerritBurgerDev commented:
> I modified the nav bar myself a bit along with the grid layout in my front-end-auth branch since I removed the login page: If we only gona have google auth then no page need. I refactored index.html to include the navbar and allowed it to be passed a slot which will be placed at the end of the nav bar. At the moment the google sign in button and welcome message (for logged in users) are set there. I was thinking we could get my front-end-auth branch merged first and the just update your nav-bar branch from master and realllign the new navbar styling you implement. Maybe even place the search bar you made in the nav in the center?

I'd prefer to leave the search bar where it is since the filters will be below it... but we can merge your branch first
